restcomm/restcomm.testsuite/src/test/java/org/mobicents/servlet/restcomm/http/CreateClientsTool.java

```diff
@@ -40,6 +48,70 @@ private String getEndpoint(String deploymentUrl) {
         return deploymentUrl;
     }
 
+    private String getAuthorizationToken(String username, String password) {
+        byte[] usernamePassBytes = (username + ":" + password).getBytes(Charset.forName("UTF-8"));
+        String authenticationToken = Base64.encodeBase64String(usernamePassBytes);
+        return authenticationToken;
+    }
+
+    private String getClientUrl(String deploymentUrl, JsonObject account) {
+        return getClientUrl(deploymentUrl, account, false);
+    }
+
+    private String getClientUrl(String deploymentUrl, JsonObject account, Boolean xml) {
+      if (deploymentUrl.endsWith("/")) {
+          deploymentUrl = deploymentUrl.substring(0, deploymentUrl.length() - 1);
+      }
+      StringBuffer curlCommand = new StringBuffer("http://");
+      curlCommand.append(account.get("sid"));
+      curlCommand.append(":");
+      curlCommand.append(account.get("auth_token"));
+      curlCommand.append(deploymentUrl.replace("http://", "@"));
+      curlCommand.append("/2012-04-24/Accounts/").append(account.get("sid"));
+
+      if(xml){
+          curlCommand.append("/Clients");
+      } else {
+          curlCommand.append("/Clients.json");
+      }
+
+      return curlCommand.toString().replace("\"", "");
+    }
+
+    public JsonObject getClientOfAccount(String deploymentUrl, JsonObject account) {
+        String url = getClientUrl(deploymentUrl, account);
+        JsonObject jsonResponse = null;
+        String authToken = getAuthorizationToken(account.get("email_address").toString(),
+                                                   account.get("auth_token").toString());
+        System.out.println("***************************************************************");
```

Comment:
Even though not so important here since this is used in testing only, it would be better to remove sysout statements and replace them with logging if needed